app/src/ui/octicons/octicons.generated.ts

```diff
@@ -1,1579 +1,348 @@
 /*
- * This file is automatically generated by the generate-octicons tool.
- * Manually changing this file will only lead to sadness.
- */
+* This file is automatically generated by the generate-octicons tool.
```

Comment:
I think prettier or some formatter was run on the last version of this file. Not terribly important but a nice to have and that way don't have to put an exception in the prettier file since this doesn't get generated very often.

Reply:
@tidy-dev With prettier formatting this file ends up at over 5000 lines. I think with one type definition per line the file ends up still very readable, at least for finding the line that defines a specific type.

```typescript
export const zoomOut: OcticonSymbolVariants = {
  '16': {
    p: [
      'M4.5 6.75h6a.75.75 0 0 1 0 1.5h-6a.75.75 0 0 1 0-1.5Z',
      'M0 7.5a7.5 7.5 0 1 1 13.307 4.747l2.473 2.473a.749.749 0 1 1-1.06 1.06l-2.473-2.473A7.5 7.5 0 0 1 0 7.5Zm7.5-6a6 6 0 1 0 0 12 6 6 0 0 0 0-12Z',
    ],
    h: 16,
    w: 16,
  },
  '24': {
    p: [
      'M14.5 11.25a.75.75 0 0 0 0-1.5h-8a.75.75 0 0 0 0 1.5h8Z',
      'M0 10.5C0 4.701 4.701 0 10.5 0S21 4.701 21 10.5c0 2.63-.967 5.033-2.564 6.875l4.344 4.345a.749.749 0 1 1-1.06 1.06l-4.345-4.344A10.459 10.459 0 0 1 10.5 21C4.701 21 0 16.299 0 10.5Zm10.5-9a9 9 0 0 0-9 9 9 9 0 0 0 9 9 9 9 0 0 0 9-9 9 9 0 0 0-9-9Z',
    ],
    h: 24,
    w: 24,
  },
```

vs.

```typescript
export const zoomOut: OcticonSymbolVariants = {"16":{"p":["M4.5 6.75h6a.75.75 0 0 1 0 1.5h-6a.75.75 0 0 1 0-1.5Z","M0 7.5a7.5 7.5 0 1 1 13.307 4.747l2.473 2.473a.749.749 0 1 1-1.06 1.06l-2.473-2.473A7.5 7.5 0 0 1 0 7.5Zm7.5-6a6 6 0 1 0 0 12 6 6 0 0 0 0-12Z"],"h":16,"w":16},"24":{"p":["M14.5 11.25a.75.75 0 0 0 0-1.5h-8a.75.75 0 0 0 0 1.5h8Z","M0 10.5C0 4.701 4.701 0 10.5 0S21 4.701 21 10.5c0 2.63-.967 5.033-2.564 6.875l4.344 4.345a.749.749 0 1 1-1.06 1.06l-4.345-4.344A10.459 10.459 0 0 1 10.5 21C4.701 21 0 16.299 0 10.5Zm10.5-9a9 9 0 0 0-9 9 9 9 0 0 0 9 9 9 9 0 0 0 9-9 9 9 0 0 0-9-9Z"],"h":24,"w":24}}
```

Reply:
Hmmm.. I am going back and forth. I think I still favor the formatted view because I can quickly see what is sizes are available for a given Octicon (which if I was placing one and realized the size isn't what I needed, I may search this file just to see the definition of that ociton). Secondly, having it formatted shows me what the type structure is for understanding how this code works at a glance. I don't really see this file as one I would scan for much. For example, if I needed to find a new Octicon, I typically look at the primer website were I can visually see the octicons to select from.